examples/c/udp-ping-pong.c

```diff
@@ -161,7 +161,14 @@ static void client(int argc,
                    unsigned max_iterations)
 {
     int sockqd = -1;
+
+    #ifdef _WIN32
+    char expected_buf[DATA_SIZE];
+    #endif
+
+    #ifdef  __linux__
     char expected_buf[data_size];
+    #endif
```

Comment:
What is the rationale for making this different?

Reply:
The MSVC compiler does not allow using non-constant array length specifications, so I used the constant.